app/packages/utilities/src/fetch.ts

```diff
@@ -33,24 +33,24 @@ export const getFetchHeaders = () => {
 
 export const getFetchOrigin = () => {
   // window is not defined in the web worker
-  if (typeof window !== "undefined" && window.FIFTYONE_SERVER_ADDRESS) {
+  if (hasWindow && window.FIFTYONE_SERVER_ADDRESS) {
     return window.FIFTYONE_SERVER_ADDRESS;
   }
+
   return fetchOrigin;
 };
 export function getFetchPathPrefix(): string {
   // window is not defined in the web worker
-  if (
-    typeof window !== "undefined" &&
-    typeof window.FIFTYONE_SERVER_PATH_PREFIX === "string"
-  ) {
+  if (hasWindow && typeof window.FIFTYONE_SERVER_PATH_PREFIX === "string") {
     return window.FIFTYONE_SERVER_PATH_PREFIX;
   }
 
-  return (
-    new URL(window.location.toString()).searchParams.get("proxy") ||
-    fetchPathPrefix
```

Comment:
Is fetchPathPrefix set anywhere/is `fetchPathPrefix === ""` always?

Reply:
https://github.com/voxel51/fiftyone/blob/develop/app/packages/utilities/src/fetch.ts#L71

It depends on if it's a worker thread requesting fetch or main thread.